tests/test_core.py

```diff
@@ -0,0 +1,40 @@
+import pendulum
+import pytest
+from singer_sdk.testing import get_tap_test_class, suites
+from singer_sdk.testing.templates import TapTestTemplate
+from test_replication_key import (
+    TABLE_NAME,
+    TapTestReplicationKey,
+    setup_test_table,
```

Comment:
setup_test_table, and teardown_test_table seem to make more sense in the `test_core` file. Just seems odd to be in replication_key